ow_lander/scripts/activity_full_digging_traj.py

```diff
@@ -36,7 +36,7 @@ def arg_parsing_lin(req):
     # Default trenching values
```

Comment:
This comment applies to default values in _all_ lander services.  I have not understood why default values are provided at this low level at all.   I don't think there should be any default values at this level, nor any "use_defaults" argument (which I do not expose in autonomy).   I realize this would be a pervasive change and I'm not suggesting it for now.

If we _do_ provide defaults in services, it would be nice to know what they mean -- this should be in both code comments and user documentation.

Reply:
I agree

Reply:
Although I don't think there is any meaning beyond these default values. I think they were defined merely to help with testing. Such that the caller wouldn't have to remember these values every time. Clearly there are better ways to deal with this.

Reply:
btw, there is already a jira ticket for getting rid of **use_defaults**, https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-447.

Additionally, I have create another related ticket https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-483 to get rid of the delete_prev_proj our API.

I can implement this within a day but I am not sure if this of a priority to us at the current moment.